src/Python-Windows/chevahbs

```diff
@@ -32,4 +32,4 @@ echo "## Extracting ZIP archive to ${INSTALL_DIR}... ##"
 execute unzip -q "$ARCHIVE" -d "${INSTALL_DIR}"
 
 echo "## Adding site-packages to python*._pth file... ##"
-echo "lib\\site-packages" >> ${INSTALL_DIR}/python38._pth
+echo "lib\\site-packages" >> ${INSTALL_DIR}/python310._pth
```

Comment:
I don't this this is needed.

The `lib/site-packages` should be added by default

Reply:
Looking at those backslashes, maybe it's only needed on Windows?